protocol.go

```diff
@@ -1,11 +1,12 @@
+// Package proxoproto implements a net.Listener supporting HAProxy PROTO protocol.
```

Comment:
I have no intention of fixing this. You can take this broken PR and do with it as you please.

Or take this PR and fix the typo later.